JitsiConference.js

```diff
@@ -2106,16 +2098,7 @@ JitsiConference.prototype._startP2PSession = function(peerJid) {
     // immediately once the P2P ICE connects.
     const localTracks = this.getLocalTracks();
 
-    logger.info(`Adding ${localTracks} to P2P...`);
-    this.p2pJingleSession.addLocalTracks(localTracks).then(
-        () => {
-            logger.info(`Added ${localTracks} to P2P`);
-            logger.info('About to send P2P \'session-initiate\'...');
-            this.p2pJingleSession.invite();
-        },
-        error => {
-            logger.error(`Failed to add ${localTracks} to P2P`, error);
-        });
+    this.p2pJingleSession.invite(localTracks);
```

Comment:
why change this from the previous method where we add the tracks first, and then do the invite?